x-pack/platform/plugins/shared/alerting/common/routes/maintenance_window/external/apis/create/schemas/v1.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+// TODO schedule schema
+const scheduleSchema = schema.object({
+  duration: schema.number(),
+  start: schema.string(),
+  recurring: schema.maybe(
+    schema.object({
+      end: schema.maybe(schema.string()),
+      every: schema.maybe(schema.string()),
+      onWeekDay: schema.maybe(schema.arrayOf(schema.string())),
+      onMonthDay: schema.maybe(schema.arrayOf(schema.number())),
+      onMonth: schema.maybe(schema.arrayOf(schema.string())),
+      occurrences: schema.maybe(schema.number()),
+    })
+  ),
+});
+
+export const bodySchema = schema.object({
```

Comment:
Should we also include the `enabled` field in the create API? Would it make sense for a user to create a maintenance window that is disabled by default?

Reply:
I would guess yes, that it may be useful.  We allow rules to be created in a disabled state.  A customer might want some set of MW's setup via terraform or similar, and then be able to enable them manually later.

Reply:
But we could also add this as a follow-on PR ...

Reply:
>MW's setup via terraform or similar, and then be able to enable them manually later.
Good point.